libcaf_net/caf/net/socket_manager.hpp

```diff
@@ -52,6 +52,11 @@ class socket_manager : public ref_counted {
     return handle_;
   }
 
+  /// Returns a pointer to the multiplexer running the socket manager.
```

Comment:
This upgrades a weak pointer to a strong pointer. So the documentation should at least mention that the function can `nullptr` and why.